### ISSUE TITLE
random Nausea effect

### ISSUE THREAD

**Issue by Vashta** [DarkPacks/SevTech-Ages#4543]
## Bug Description

was in my base on an entirely lit up area when out of no where i got a 4 second nausea i went outside my house and looked around but no random enemies that could possibly be the cause this and went back in only to have it happen now 3 times after the first time so basically i got nausea for no determinable reason so if anyone knows please tell me the causes for nausea in here 

### Did this behavior use to work in the previous version?

this is my first version playing on

## Steps to Reproduce (for bugs)

1. tried multiple ways cant reliably get it to happen again
2. 
3.

## Logs

* Client/Server Log:
* Crash Log:

## Client Information
* Modpack Version: current
* Java Version: current
* Launcher Used:
* Memory Allocated:
* Server/LAN/Single Player:
* Optifine Installed:
* Shaders Enabled:

## World Information
* Modpack Version world created in:
* Additional Content Installed:

**Comment by sam-kirby** [comment]
Have you started progressing with blood magic?

**Comment by Vashta** [comment]
> 
> 
> Have you started progressing with blood magic?

yes i did see others mentioning the lava crystal but ive never made one so it couldnt have been that

**Comment by Vashta** [comment]
> 
> 
> Have you started progressing with blood magic?

ive noticed i get it a bit more often while in the very center of my base inside where i have a couple burning coal blocks inside there isnt some weird thing where the smoke particles matter would it? if so why did it appear after era 2 only?

**Comment by sam-kirby** [comment]
Smoke particles are a visual effect only as far as I'm aware.

I'm not just asking about a lava crystal though, lots of things in blood magic can cause a nausea effect, if you have *any* rituals check you have LP in your blood network using a sigil of divination.
